src/plays/markdown-editor/styles.css

```diff
@@ -0,0 +1,36 @@
+/* enter stlyes here */
+:root {
```

Comment:
Please do not override. These selectors are already overridden at the framework root level. So remove.